compiler/src/iree/compiler/DispatchCreation/FuseHorizontalContractions.cpp

```diff
@@ -171,52 +179,25 @@ getTruncateOp(Operation *op,
 /// %4 = linalg.matmul ins(%arg0, concat(%arg1, %arg2, %arg3))
 /// ```
 ///
-/// This method recognizes such patterns. It also accounts for the quantized
-/// case where individual operations might be have lower-precision operands and
-/// accumulate in higher precision, followed by a `linalg.generic` that performs
-/// the `truncf` on the result.
-static std::optional<HorizontalFusionGroup> getHorizontalFusionGroupMembers(
+/// Note: The actual operation generated does not concat the RHS
```

Comment:
nit: add a period at the end to finish the sentence

```suggestion
/// Note: The actual operation generated does not concat the RHS.
```